server/libs/infra/src/entities/person.entity.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { UserEntity } from './user.entity';
+import { AssetFaceEntity } from './asset-face.entity';
+
+@Entity('person')
+export class PersonEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string;
+
+  @Column()
+  ownerId!: string;
+
+  @Column()
+  name!: string;
+
+  @Column()
+  thumbnailPath!: string;
+
+  @Column({
+    type: 'float4',
+    array: true,
+    nullable: true,
+  })
+  embedded!: number[] | null;
```

Comment:
I don't think we need this stored on person, do we?

Reply:
Do we need it to compare faces when an asset is uploaded?

Reply:
We need to store the embeddings, yes, but on the faces table. We could store a single, "central", embedding per person to use in calculations, but that's not the clustering approach we're going for. Instead we're going to try using typesense to find nearby faces and then use that data to assign the new face to a person.